backend/danswer/server/admin.py

```diff
@@ -140,3 +149,59 @@ def list_all_index_attempts(
             for index_attempt in index_attempts
         ]
     )
+
+
+@router.head("/kv/openai_api_key/validate")
```

Comment:
preference for kv_store over kv which could also mean keyvault which this very much is not

Reply:
or better yet, key_val_store